randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -189,9 +187,11 @@ Extra - asset_id: collision_camera_003
 > Center Platform; Heals? False
   * Layers: default
   > Dock to Teleport to Burenia (Cyan)
-      Grapple Beam or Gravity Suit or Use Spin Boost
+      Flash Shift or Grapple Beam or Gravity Suit or Spider Magnet or Lay Cross Comb
```

Comment:
I would change Lay Cross Comb to Simple IBJ, and then add Grapple Movement to Grapple Beam. Also curious why you removed Use Spin Boost

Reply:
With IBJ you barely come out of the water and would need SM to climb out. With Cross bombs you can just jump out. I would say IBJ is not enough

Reply:
So do it from the platform?